pkg/controller/pravega/constants.go

```diff
@@ -19,6 +19,8 @@ const (
 	ssSecretVolumeName     = "ss-secret"
 	tlsVolumeName          = "tls-secret"
 	tlsMountDir            = "/etc/secret-volume"
+	caBundleVolumeName     = "ca-bundle"
+	caBundleMountDir       = "/etc/secret-volume/ca-bundle"
```

Comment:
I noticed this is in `/etc/secret-volume` which is also used by `tlsMountDir`.  If both options are provided, is it safe to mount a volume as a subset of another volume mount?

Reply:
Yes I've tested all possibilities:
SS TLS only, 
ECS TLS only,
SS + ECS 
none
all worked.